gradle/checkstyle/header.txt

```diff
@@ -0,0 +1,2 @@
+^/\*
+^ \* (Licensed to|Copyright|This file is part of a module with proprietary Enterprise Features)
```

Comment:
Shouldn't we include the whole header? or at least a line count with the comment?

Reply:
If you check out the commit message, it says 

```
This performs a basic check if there is some sort of license header in
place. Checking for the exact header is rather cumbersome because we use
different headers and the Checkstyle plugin only allows to check the header line
by line.
```

I would keep it as simple as that because it is not really possible to fully check different headers with the functionality of the Checkstyle plugin. It is rather designed to just check a single license.

Reply:
Ah yep, I see sorry didn't read the comments carefully.

If not too troublesome would be nice to be able to distinguish the Enterprise Licence for the enterprise modules.

Reply:
That's possible and quite nice to have.